manifests/namespace-install/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ bases:
 - ../crds
 - ../base
 - ../role
+- argo-rollouts-rolebinding.yaml
```

Comment:
@tdongsi this should actually be under resources.

Also, you will need to run `make manifests` after the kustomization.yaml change.